src/components/atoms/Functions.tsx

```diff
@@ -0,0 +1,18 @@
+import { useEffect, useState } from "react";
+
+  const usePersistedState = (key:string,value:unknown) => {
+    const [state, setState] = useState(() => {
+      const storedValue = localStorage.getItem(key);
+      return storedValue ? JSON.parse(storedValue) : value;
+    });
+  
+    useEffect(() => {
+      localStorage.setItem(key, JSON.stringify(state));
+    }, [state, key]);
+  
+    return [state, setState] as const;
+  }
+
+
+
```

Comment:
Always format your code before pushing to origin

Reply:
> Always format your code before pushing to origin

Sure.